frontend/src/v4/routes/board/board.component.tsx

```diff
@@ -260,9 +261,12 @@ export function Board(props: IProps) {
 	};
 
 	useEffect(() => {
-		if (isV5() && boardRef.current) {
+		if (isV5() && boardRef.current && !firstUpdate.current) {
```

Comment:
The firstUpdate logic was needed to prevent the board instantly resetting when clicking a link to an issues/risks kanban